lib/phoenix/test/conn_test.ex

```diff
@@ -354,7 +354,7 @@ defmodule Phoenix.ConnTest do
     if given == status do
       body
     else
-      raise "expected response with status #{given}, got: #{status}"
+      raise "expected response with status #{given}, got: #{status}, for: #{body}"
```

Comment:
Maybe put the body on a new line? I bet it can be pretty big and we dont want the actual error message lost in the noise.